CRM/Contact/BAO/Query.php

```diff
@@ -4751,8 +4751,8 @@ public static function getGroupByFromSelectColumns($selectClauses, $groupBy = NU
     $mysqlVersion = CRM_Core_DAO::singleValueQuery('SELECT VERSION()');
     $sqlMode = CRM_Core_DAO::singleValueQuery('SELECT @@sql_mode');
 
-    //return if ONLY_FULL_GROUP_BY is not enabled.
-    if (CRM_Utils_SQL::supportsFullGroupBy() && !empty($sqlMode) && in_array('ONLY_FULL_GROUP_BY', explode(',', $sqlMode))) {
+    // If ONLY_FULL_GROUP_BY is enabled, add fields to group by statement.
+    if (!empty($sqlMode) && in_array('ONLY_FULL_GROUP_BY', explode(',', $sqlMode))) {
       $regexToExclude = '/(ROUND|AVG|COUNT|GROUP_CONCAT|SUM|MAX|MIN|IF)[[:blank:]]*\(/i';
```

Comment:
This change is a bit terrifying, but CRM_Utils_SQL::supportsFullGroupBy is deprecated and, in addition to checking if a SQL server is configured to use ONLY_FULL_GROUP_BY it also checks if the server supports ANY_VALUE, which mariadb doesn't, so it always returns FALSE. I don't think this function depends on ANY_VALUE, so I'm simplifying the check to only determine if ONLY_FULL_GROUP_BY is in place.